src/pages/filter/dtree/components/modals/components/create-dataset-dialog.tsx

```diff
@@ -0,0 +1,203 @@
+import { useEffect, useState } from 'react'
+import { useHistory } from 'react-router-dom'
+import { toJS } from 'mobx'
+import { observer } from 'mobx-react-lite'
+
+import { DatasetCreationErrorsEnum } from '@core/enum/dataset-creation-errors-enum'
+import { PatnNameEnum } from '@core/enum/path-name-enum'
+import { t } from '@i18n'
+import datasetStore from '@store/dataset/dataset'
+import dirinfoStore from '@store/dirinfo'
+import dtreeStore from '@store/dtree'
+import filterStore from '@store/filter'
+import filterPresetsStore from '@store/filter-presets'
+import operations from '@store/operations'
+import mainTableStore from '@store/ws/main-table.store'
+import zoneStore from '@store/ws/zone'
+import { Routes } from '@router/routes.enum'
+import { Dialog } from '@ui/dialog'
+import { IBaseDialogProps } from '@ui/dialog/dialog.interface'
+// TODO: convert attention icon to project icon format
+import { Attention } from '@ui/icon/icons/attention'
+import { Input } from '@ui/input'
+import { DecisionTreesMenuDataCy } from '@components/data-testid/decision-tree-menu.cy'
+import { GlbPagesNames } from '@glb/glb-names'
+import { showToast } from '@utils/notifications/showToast'
+import {
+  noFirstNumberPattern,
+  noSymbolPattern,
+} from '@utils/validation/validationPatterns'
+
+export const CreateDatasetDialog = observer(
```

Comment:
Input and create button should be disabled while dataset is creating, it was done in previous realization, but you lost it, while moving file